python/BioSimSpace/Sandpit/Exscientia/Process/_amber.py

```diff
@@ -926,11 +966,18 @@ def getRecords(self, dof=0, block="AUTO"):
         """
 
         # Validate the degree of freedom.
-        if not isinstance(dof, int):
-            raise TypeError("'dof' must be of type 'int'")
+        if not isinstance(region, int):
+            raise TypeError("'region' must be of type 'int'")
         else:
-            if dof < 0 or dof > 2:
-                raise ValueError("'dof' must be in range [0, 2]")
+            if region < 0 or region > 1:
```

Comment:
Why not just check `region in (0,1)`?

Reply:
Could do, but this is faster since it would exit if the first conditional was `True`, whereas the `in` membership test approach needs to iterate over all items in the tuple and check each one. There is also the overhead of creating the tuple, iterating over it. Obviously speed is pretty irrelevant for something this small, I'm just used to writing conditionals so that they are as efficient as possible.  (I'd certainly use `in` for a list of options, though.)